x/txfees/keeper/txfee_filters/arb_tx.go

```diff
@@ -35,7 +36,10 @@ func IsArbTxLoose(tx sdk.Tx) bool {
 
 		swapMsg, isSwapMsg := m.(gammtypes.SwapMsgRoute)
 		if !isSwapMsg {
-			continue
+			swapMsg, isSwapMsg = m.(poolmanagertypes.SwapMsgRoute)
```

Comment:
Sorry I don't quite follow here, doesn't gammtypes.SwapMsgRoute and poolmanagertypes.SwapMsgRoute have the same interfaces and methods?
What would be the case that msg here fails type conversion for gammtypes.SwapMsgRoute but suceed in type casting to poolmanagertypes.SwapMsgRoute?